src/main/java/com/google/sps/data/Response.java

```diff
@@ -44,8 +44,7 @@ public String getStatus() {
     public void pick() {
         if(results.size() > 0){
             int randIdx = (results.size() > 1) ? new Random().nextInt(results.size() - 1) : 0;
-            pick = results.get(randIdx);
-            results.remove(randIdx);
+            this.pick = results.get(randIdx); results.remove(randIdx);
```

Comment:
formatting

Sorry I did have the formatting correct in the issue I opened

Reply:
done